server/src/main/java/org/elasticsearch/rest/RestRequest.java

```diff
@@ -306,9 +303,27 @@ public boolean isFullContent() {
         return httpRequest.body().isFull();
     }
 
+    /**
+     * Returns a copy of HTTP content. The copy is GC-managed and does not require reference counting.
+     * Please use {@link #releasableContent()} to avoid content copy.
+     */
     public BytesReference content() {
+        return BytesReference.copyBytes(releasableContent());
+    }
+
+    /**
+     * Returns a reference to the network buffer of HTTP content.
```

Comment:
```suggestion
     * Returns a direct reference to the network buffer containing the request body. The HTTP layers will release their references to this
     * buffer as soon as they have finished the synchronous steps of processing the request on the network thread, which will by default
     * release the buffer back to the pool where it may be re-used for another request. If you need to keep the buffer alive past the end of
     * these synchronous steps, acquire your own reference to this buffer and release it once it's no longer needed.
```

Reply:
Applied suggestion.